huxley/api/views/school.py

```diff
@@ -68,9 +69,9 @@ def get(self, request, *args, **kwargs):
             "date_registered": school.registered.strftime("%m/%d/%y"),
             "due_date": due_date.strftime("%m/%d/%y"),
             "delegate_total": delegate_total,
-            "delegate_fee": School.DELEGATE_FEE,
+            "delegate_fee": delegate_fee,
             "delegate_fees": delegate_fees,
-            "registration_fee": School.REGISTRATION_FEE,
+            "registration_fee": Conference.get_conference().registration_fee,
```

Comment:
We should assign to a local variable if we're gonna be using the conference multiple times :)